config/local/kustomization.yaml

```diff
@@ -0,0 +1,21 @@
+# Adds namespace to all resources.
+namespace: perses-operator-system
+
+namePrefix: perses-operator-
+
+resources:
+  - ../crd/
+  - ../rbac
+  - ../manager
+  - ../webhook
+  - certificate.yaml
+
+patchesStrategicMerge:
+  - patches/manager_webhook_patch.yaml
```

Comment:
Tried locally and this file does not exist. Is it the right path? I see it under `default/manager_webhook_patch.yaml`